tests/test_events.py

```diff
@@ -53,32 +53,32 @@ def test_rrule():
             '''
         )
     )
-    assert 'DTEND' not in event
-    assert 'RRULE:FREQ=YEARLY;UNTIL=20300422T000000Z' in event
+    assert 'DTEND' not in event.serialize()
+    assert 'RRULE:FREQ=YEARLY;UNTIL=20300422T000000' in event.serialize()
 
 
 def test_event_with_time_range():
     event = event_ics_from_yaml(
         parse_yaml(
             '''
             name: Event of the Century
-            begin: 2021-09-21 15:00-07:00
-            end: 2021-09-21 15:30:00-07:00
+            begin: 2021-09-21 15:00:00 -07:00
```

Comment:
Is this also an improvement from the new ics?  I like it!

Reply:
yes (required for native yaml parsing of datetimes)